libraries/lib-module-manager/PluginManager.h

```diff
@@ -174,9 +37,46 @@ typedef wxArrayString PluginIDs;
 
 class PluginRegistrationDialog;
 
+class MODULE_MANAGER_API PluginValidator
```

Comment:
Four new abstract classes are introduced in the commit "PluginValidator interface" but I'm not sure what they really do, or how they are meant to be used, from reading their names alone and from the lack of comments.  I must study other code to understand.